.github/workflows/ci-core.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           allow-extra-out-format-args: true
           args: --out-format checkstyle:golangci-lint-report.xml
       - name: Print lint report artifact
-        if: always()
+        if: needs.init.outputs.on_trigger_lint == 'true'
```

Comment:
@jmank88 This fails if there is no go changes to lint so I made it match the above if

Reply:
@tateexon in case we need `always()` what if we did https://github.com/smartcontractkit/chainlink/pull/8808/files#r1146925806 instead?

Reply:
This trigger is insufficient when the linter detects issues.